diff --color -ruN nginx-1.26.2/conf/fastcgi.conf nginx-1.26_changes/conf/fastcgi.conf
--- nginx-1.26.2/conf/fastcgi.conf	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/conf/fastcgi.conf	2024-09-18 05:02:27.349397576 +0200
@@ -14,7 +14,6 @@
 fastcgi_param  HTTPS              $https if_not_empty;
 
 fastcgi_param  GATEWAY_INTERFACE  CGI/1.1;
-fastcgi_param  SERVER_SOFTWARE    nginx/$nginx_version;
 
 fastcgi_param  REMOTE_ADDR        $remote_addr;
 fastcgi_param  REMOTE_PORT        $remote_port;
diff --color -ruN nginx-1.26.2/conf/fastcgi_params nginx-1.26_changes/conf/fastcgi_params
--- nginx-1.26.2/conf/fastcgi_params	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/conf/fastcgi_params	2024-09-18 05:02:46.879171195 +0200
@@ -13,7 +13,6 @@
 fastcgi_param  HTTPS              $https if_not_empty;
 
 fastcgi_param  GATEWAY_INTERFACE  CGI/1.1;
-fastcgi_param  SERVER_SOFTWARE    nginx/$nginx_version;
 
 fastcgi_param  REMOTE_ADDR        $remote_addr;
 fastcgi_param  REMOTE_PORT        $remote_port;
diff --color -ruN nginx-1.26.2/html/50x.html nginx-1.26_changes/html/50x.html
--- nginx-1.26.2/html/50x.html	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/html/50x.html	2024-09-18 04:58:05.299445578 +0200
@@ -1,19 +1,65 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Error</title>
+<title>Error: 50x</title>
 <style>
-html { color-scheme: light dark; }
-body { width: 35em; margin: 0 auto;
-font-family: Tahoma, Verdana, Arial, sans-serif; }
+html { 
+    color-scheme: light dark; 
+}
+
+body { 
+    width: 35em; 
+    margin: 0 auto;
+    font-family: Tahoma, Verdana, Arial, sans-serif;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100vh;
+}
+
+span {
+    font-family: sans-serif;
+    white-space: normal;
+    max-width: 90%;
+    display: block;
+    margin: 0 auto;
+}
+
+span.big-text {
+    font-size: 100px;
+    font-weight: bold;
+    margin-bottom: 20px;
+}
+
+span.small-text {
+    font-size: 40px;
+    font-weight: 100;
+}
+
+@media (max-width: 768px) {
+    span.big-text {
+        font-size: 70px;
+    }
+    span.small-text {
+        font-size: 30px;
+    }
+}
+
+@media (max-width: 480px) {
+    span.big-text {
+        font-size: 50px;
+    }
+    span.small-text {
+        font-size: 20px;
+    }
+}
 </style>
 </head>
 <body>
-<h1>An error occurred.</h1>
-<p>Sorry, the page you are looking for is currently unavailable.<br/>
-Please try again later.</p>
-<p>If you are the system administrator of this resource then you should check
-the error log for details.</p>
-<p><em>Faithfully yours, nginx.</em></p>
+
+<span class="big-text">50x</span>
+<span class="small-text">An error occurred</span>
+
 </body>
 </html>
diff --color -ruN nginx-1.26.2/html/index.html nginx-1.26_changes/html/index.html
--- nginx-1.26.2/html/index.html	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/html/index.html	2024-09-18 04:40:05.050595075 +0200
@@ -1,23 +1,13 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Welcome to nginx!</title>
+<title>Server is up and running!</title>
 <style>
 html { color-scheme: light dark; }
-body { width: 35em; margin: 0 auto;
-font-family: Tahoma, Verdana, Arial, sans-serif; }
+body { font-family: sans-serif; }
 </style>
 </head>
 <body>
-<h1>Welcome to nginx!</h1>
-<p>If you see this page, the nginx web server is successfully installed and
-working. Further configuration is required.</p>
-
-<p>For online documentation and support please refer to
-<a href="http://nginx.org/">nginx.org</a>.<br/>
-Commercial support is available at
-<a href="http://nginx.com/">nginx.com</a>.</p>
-
-<p><em>Thank you for using nginx.</em></p>
+<p>Server is up and running.</p>
 </body>
 </html>
diff --color -ruN nginx-1.26.2/src/core/nginx.c nginx-1.26_changes/src/core/nginx.c
--- nginx-1.26.2/src/core/nginx.c	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/src/core/nginx.c	2024-09-18 05:16:42.891730981 +0200
@@ -391,7 +391,7 @@
 static void
 ngx_show_version_info(void)
 {
-    ngx_write_stderr("nginx version: " NGINX_VER_BUILD NGX_LINEFEED);
+    ngx_write_stderr("nginx version: nginx 1.26.2" NGX_LINEFEED);
 
     if (ngx_show_help) {
         ngx_write_stderr(
diff --color -ruN nginx-1.26.2/src/core/nginx.h nginx-1.26_changes/src/core/nginx.h
--- nginx-1.26.2/src/core/nginx.h	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/src/core/nginx.h	2024-09-18 05:08:09.025447469 +0200
@@ -10,8 +10,8 @@
 
 
 #define nginx_version      1026002
-#define NGINX_VERSION      "1.26.2"
-#define NGINX_VER          "nginx/" NGINX_VERSION
+#define NGINX_VERSION      ""
+#define NGINX_VER          "" NGINX_VERSION
 
 #ifdef NGX_BUILD
 #define NGINX_VER_BUILD    NGINX_VER " (" NGX_BUILD ")"
diff --color -ruN nginx-1.26.2/src/http/ngx_http_header_filter_module.c nginx-1.26_changes/src/http/ngx_http_header_filter_module.c
--- nginx-1.26.2/src/http/ngx_http_header_filter_module.c	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/src/http/ngx_http_header_filter_module.c	2024-09-18 06:43:31.648704886 +0200
@@ -46,9 +46,7 @@
 };
 
 
-static u_char ngx_http_server_string[] = "Server: nginx" CRLF;
-static u_char ngx_http_server_full_string[] = "Server: " NGINX_VER CRLF;
-static u_char ngx_http_server_build_string[] = "Server: " NGINX_VER_BUILD CRLF;
+static u_char ngx_http_server_string[] = "" CRLF;
 
 
 static ngx_str_t ngx_http_status_lines[] = {
@@ -284,15 +282,9 @@
     clcf = ngx_http_get_module_loc_conf(r, ngx_http_core_module);
 
     if (r->headers_out.server == NULL) {
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            len += sizeof(ngx_http_server_full_string) - 1;
 
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            len += sizeof(ngx_http_server_build_string) - 1;
-
-        } else {
             len += sizeof(ngx_http_server_string) - 1;
-        }
+        
     }
 
     if (r->headers_out.date == NULL) {
@@ -453,18 +445,10 @@
     *b->last++ = CR; *b->last++ = LF;
 
     if (r->headers_out.server == NULL) {
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            p = ngx_http_server_full_string;
-            len = sizeof(ngx_http_server_full_string) - 1;
-
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            p = ngx_http_server_build_string;
-            len = sizeof(ngx_http_server_build_string) - 1;
 
-        } else {
             p = ngx_http_server_string;
             len = sizeof(ngx_http_server_string) - 1;
-        }
+        
 
         b->last = ngx_cpymem(b->last, p, len);
     }
diff --color -ruN nginx-1.26.2/src/http/ngx_http_special_response.c nginx-1.26_changes/src/http/ngx_http_special_response.c
--- nginx-1.26.2/src/http/ngx_http_special_response.c	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/src/http/ngx_http_special_response.c	2024-09-18 06:20:51.640337563 +0200
@@ -18,34 +18,20 @@
 static ngx_int_t ngx_http_send_refresh(ngx_http_request_t *r);
 
 
-static u_char ngx_http_error_full_tail[] =
-"<hr><center>" NGINX_VER "</center>" CRLF
-"</body>" CRLF
-"</html>" CRLF
-;
-
-
-static u_char ngx_http_error_build_tail[] =
-"<hr><center>" NGINX_VER_BUILD "</center>" CRLF
-"</body>" CRLF
-"</html>" CRLF
-;
-
-
 static u_char ngx_http_error_tail[] =
-"<hr><center>nginx</center>" CRLF
 "</body>" CRLF
 "</html>" CRLF
 ;
 
 
 static u_char ngx_http_msie_padding[] =
-"<!-- a padding to disable MSIE and Chrome friendly error page -->" CRLF
-"<!-- a padding to disable MSIE and Chrome friendly error page -->" CRLF
-"<!-- a padding to disable MSIE and Chrome friendly error page -->" CRLF
-"<!-- a padding to disable MSIE and Chrome friendly error page -->" CRLF
-"<!-- a padding to disable MSIE and Chrome friendly error page -->" CRLF
-"<!-- a padding to disable MSIE and Chrome friendly error page -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
+"<!-- padding padding padding padding padding padding padding padding -->" CRLF
 ;
 
 
@@ -59,284 +45,653 @@
 
 static char ngx_http_error_301_page[] =
 "<html>" CRLF
-"<head><title>301 Moved Permanently</title></head>" CRLF
+"<head>" CRLF
+"<title>301</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>301 Moved Permanently</h1></center>" CRLF
+"<span class=\"big-text\">301</span>" CRLF
+"<span class=\"small-text\">Moved Permanently</span>" CRLF
 ;
 
 
 static char ngx_http_error_302_page[] =
 "<html>" CRLF
-"<head><title>302 Found</title></head>" CRLF
+"<head>" CRLF
+"<title>302</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>302 Found</h1></center>" CRLF
+"<span class=\"big-text\">302</span>" CRLF
+"<span class=\"small-text\">Found</span>" CRLF
 ;
 
 
+
 static char ngx_http_error_303_page[] =
 "<html>" CRLF
-"<head><title>303 See Other</title></head>" CRLF
+"<head>" CRLF
+"<title>303</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>303 See Other</h1></center>" CRLF
+"<span class=\"big-text\">303</span>" CRLF
+"<span class=\"small-text\">See Other</span>" CRLF
 ;
 
-
 static char ngx_http_error_307_page[] =
 "<html>" CRLF
-"<head><title>307 Temporary Redirect</title></head>" CRLF
+"<head>" CRLF
+"<title>307</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>307 Temporary Redirect</h1></center>" CRLF
+"<span class=\"big-text\">307</span>" CRLF
+"<span class=\"small-text\">Temporary Redirect</span>" CRLF
 ;
 
-
 static char ngx_http_error_308_page[] =
 "<html>" CRLF
-"<head><title>308 Permanent Redirect</title></head>" CRLF
+"<head>" CRLF
+"<title>308</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>308 Permanent Redirect</h1></center>" CRLF
+"<span class=\"big-text\">308</span>" CRLF
+"<span class=\"small-text\">Permanent Redirect</span>" CRLF
 ;
 
-
 static char ngx_http_error_400_page[] =
 "<html>" CRLF
-"<head><title>400 Bad Request</title></head>" CRLF
+"<head>" CRLF
+"<title>400</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>400 Bad Request</h1></center>" CRLF
+"<span class=\"big-text\">400</span>" CRLF
+"<span class=\"small-text\">Bad Request</span>" CRLF
 ;
 
-
 static char ngx_http_error_401_page[] =
 "<html>" CRLF
-"<head><title>401 Authorization Required</title></head>" CRLF
+"<head>" CRLF
+"<title>401</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>401 Authorization Required</h1></center>" CRLF
+"<span class=\"big-text\">401</span>" CRLF
+"<span class=\"small-text\">Authorization Required</span>" CRLF
 ;
 
-
 static char ngx_http_error_402_page[] =
 "<html>" CRLF
-"<head><title>402 Payment Required</title></head>" CRLF
+"<head>" CRLF
+"<title>402</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>402 Payment Required</h1></center>" CRLF
+"<span class=\"big-text\">402</span>" CRLF
+"<span class=\"small-text\">Payment Required</span>" CRLF
 ;
 
-
 static char ngx_http_error_403_page[] =
 "<html>" CRLF
-"<head><title>403 Forbidden</title></head>" CRLF
+"<head>" CRLF
+"<title>403</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>403 Forbidden</h1></center>" CRLF
+"<span class=\"big-text\">403</span>" CRLF
+"<span class=\"small-text\">Forbidden</span>" CRLF
 ;
 
-
 static char ngx_http_error_404_page[] =
 "<html>" CRLF
-"<head><title>404 Not Found</title></head>" CRLF
+"<head>" CRLF
+"<title>404</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>404 Not Found</h1></center>" CRLF
+"<span class=\"big-text\">404</span>" CRLF
+"<span class=\"small-text\">Not Found</span>" CRLF
 ;
 
-
 static char ngx_http_error_405_page[] =
 "<html>" CRLF
-"<head><title>405 Not Allowed</title></head>" CRLF
+"<head>" CRLF
+"<title>405</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>405 Not Allowed</h1></center>" CRLF
+"<span class=\"big-text\">405</span>" CRLF
+"<span class=\"small-text\">Not Allowed</span>" CRLF
 ;
 
-
 static char ngx_http_error_406_page[] =
 "<html>" CRLF
-"<head><title>406 Not Acceptable</title></head>" CRLF
+"<head>" CRLF
+"<title>406</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>406 Not Acceptable</h1></center>" CRLF
+"<span class=\"big-text\">406</span>" CRLF
+"<span class=\"small-text\">Not Acceptable</span>" CRLF
 ;
 
-
 static char ngx_http_error_408_page[] =
 "<html>" CRLF
-"<head><title>408 Request Time-out</title></head>" CRLF
+"<head>" CRLF
+"<title>408</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>408 Request Time-out</h1></center>" CRLF
+"<span class=\"big-text\">408</span>" CRLF
+"<span class=\"small-text\">Request Time-out</span>" CRLF
 ;
 
-
 static char ngx_http_error_409_page[] =
 "<html>" CRLF
-"<head><title>409 Conflict</title></head>" CRLF
+"<head>" CRLF
+"<title>409</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>409 Conflict</h1></center>" CRLF
+"<span class=\"big-text\">409</span>" CRLF
+"<span class=\"small-text\">Conflict</span>" CRLF
 ;
 
-
 static char ngx_http_error_410_page[] =
 "<html>" CRLF
-"<head><title>410 Gone</title></head>" CRLF
+"<head>" CRLF
+"<title>410</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>410 Gone</h1></center>" CRLF
+"<span class=\"big-text\">410</span>" CRLF
+"<span class=\"small-text\">Gone</span>" CRLF
 ;
 
-
 static char ngx_http_error_411_page[] =
 "<html>" CRLF
-"<head><title>411 Length Required</title></head>" CRLF
+"<head>" CRLF
+"<title>411</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>411 Length Required</h1></center>" CRLF
+"<span class=\"big-text\">411</span>" CRLF
+"<span class=\"small-text\">Length Required</span>" CRLF
 ;
 
-
 static char ngx_http_error_412_page[] =
 "<html>" CRLF
-"<head><title>412 Precondition Failed</title></head>" CRLF
+"<head>" CRLF
+"<title>412</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>412 Precondition Failed</h1></center>" CRLF
+"<span class=\"big-text\">412</span>" CRLF
+"<span class=\"small-text\">Precondition Failed</span>" CRLF
 ;
 
-
 static char ngx_http_error_413_page[] =
 "<html>" CRLF
-"<head><title>413 Request Entity Too Large</title></head>" CRLF
+"<head>" CRLF
+"<title>413</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>413 Request Entity Too Large</h1></center>" CRLF
+"<span class=\"big-text\">413</span>" CRLF
+"<span class=\"small-text\">Request Entity Too Large</span>" CRLF
 ;
 
-
 static char ngx_http_error_414_page[] =
 "<html>" CRLF
-"<head><title>414 Request-URI Too Large</title></head>" CRLF
+"<head>" CRLF
+"<title>414</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>414 Request-URI Too Large</h1></center>" CRLF
+"<span class=\"big-text\">414</span>" CRLF
+"<span class=\"small-text\">Request-URI Too Large</span>" CRLF
 ;
 
-
 static char ngx_http_error_415_page[] =
 "<html>" CRLF
-"<head><title>415 Unsupported Media Type</title></head>" CRLF
+"<head>" CRLF
+"<title>415</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>415 Unsupported Media Type</h1></center>" CRLF
+"<span class=\"big-text\">415</span>" CRLF
+"<span class=\"small-text\">Unsupported Media Type</span>" CRLF
 ;
 
-
 static char ngx_http_error_416_page[] =
 "<html>" CRLF
-"<head><title>416 Requested Range Not Satisfiable</title></head>" CRLF
+"<head>" CRLF
+"<title>416</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>416 Requested Range Not Satisfiable</h1></center>" CRLF
+"<span class=\"big-text\">416</span>" CRLF
+"<span class=\"small-text\">Requested Range Not Satisfiable</span>" CRLF
 ;
 
-
 static char ngx_http_error_421_page[] =
 "<html>" CRLF
-"<head><title>421 Misdirected Request</title></head>" CRLF
+"<head>" CRLF
+"<title>421</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>421 Misdirected Request</h1></center>" CRLF
+"<span class=\"big-text\">421</span>" CRLF
+"<span class=\"small-text\">Misdirected Request</span>" CRLF
 ;
 
-
 static char ngx_http_error_429_page[] =
 "<html>" CRLF
-"<head><title>429 Too Many Requests</title></head>" CRLF
+"<head>" CRLF
+"<title>429</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>429 Too Many Requests</h1></center>" CRLF
+"<span class=\"big-text\">429</span>" CRLF
+"<span class=\"small-text\">Too Many Requests</span>" CRLF
 ;
 
-
 static char ngx_http_error_494_page[] =
 "<html>" CRLF
-"<head><title>400 Request Header Or Cookie Too Large</title></head>"
-CRLF
+"<head>" CRLF
+"<title>400</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>400 Bad Request</h1></center>" CRLF
-"<center>Request Header Or Cookie Too Large</center>" CRLF
+"<span class=\"big-text\">400</span>" CRLF
+"<span class=\"small-text\">Bad Request: Request Header Or Cookie Too Large</span>" CRLF
 ;
 
-
 static char ngx_http_error_495_page[] =
 "<html>" CRLF
-"<head><title>400 The SSL certificate error</title></head>"
-CRLF
+"<head>" CRLF
+"<title>400</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>400 Bad Request</h1></center>" CRLF
-"<center>The SSL certificate error</center>" CRLF
+"<span class=\"big-text\">400</span>" CRLF
+"<span class=\"small-text\">Bad Request: The SSL certificate error</span>" CRLF
 ;
 
-
 static char ngx_http_error_496_page[] =
 "<html>" CRLF
-"<head><title>400 No required SSL certificate was sent</title></head>"
-CRLF
+"<head>" CRLF
+"<title>400</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>400 Bad Request</h1></center>" CRLF
-"<center>No required SSL certificate was sent</center>" CRLF
+"<span class=\"big-text\">400</span>" CRLF
+"<span class=\"small-text\">Bad Request: No required SSL certificate was sent</span>" CRLF
 ;
 
-
 static char ngx_http_error_497_page[] =
 "<html>" CRLF
-"<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>"
-CRLF
+"<head>" CRLF
+"<title>400</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>400 Bad Request</h1></center>" CRLF
-"<center>The plain HTTP request was sent to HTTPS port</center>" CRLF
+"<span class=\"big-text\">400</span>" CRLF
+"<span class=\"small-text\">Bad Request: The plain HTTP request was sent to HTTPS port</span>" CRLF
 ;
 
-
 static char ngx_http_error_500_page[] =
 "<html>" CRLF
-"<head><title>500 Internal Server Error</title></head>" CRLF
+"<head>" CRLF
+"<title>500</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>500 Internal Server Error</h1></center>" CRLF
+"<span class=\"big-text\">500</span>" CRLF
+"<span class=\"small-text\">Internal Server Error</span>" CRLF
 ;
 
-
 static char ngx_http_error_501_page[] =
 "<html>" CRLF
-"<head><title>501 Not Implemented</title></head>" CRLF
+"<head>" CRLF
+"<title>501</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>501 Not Implemented</h1></center>" CRLF
+"<span class=\"big-text\">501</span>" CRLF
+"<span class=\"small-text\">Not Implemented</span>" CRLF
 ;
 
-
 static char ngx_http_error_502_page[] =
 "<html>" CRLF
-"<head><title>502 Bad Gateway</title></head>" CRLF
+"<head>" CRLF
+"<title>502</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>502 Bad Gateway</h1></center>" CRLF
+"<span class=\"big-text\">502</span>" CRLF
+"<span class=\"small-text\">Bad Gateway</span>" CRLF
 ;
 
-
 static char ngx_http_error_503_page[] =
 "<html>" CRLF
-"<head><title>503 Service Temporarily Unavailable</title></head>" CRLF
+"<head>" CRLF
+"<title>503</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>503 Service Temporarily Unavailable</h1></center>" CRLF
+"<span class=\"big-text\">503</span>" CRLF
+"<span class=\"small-text\">Service Temporarily Unavailable</span>" CRLF
 ;
 
-
 static char ngx_http_error_504_page[] =
 "<html>" CRLF
-"<head><title>504 Gateway Time-out</title></head>" CRLF
+"<head>" CRLF
+"<title>504</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>504 Gateway Time-out</h1></center>" CRLF
+"<span class=\"big-text\">504</span>" CRLF
+"<span class=\"small-text\">Gateway Time-out</span>" CRLF
 ;
 
-
 static char ngx_http_error_505_page[] =
 "<html>" CRLF
-"<head><title>505 HTTP Version Not Supported</title></head>" CRLF
+"<head>" CRLF
+"<title>505</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>505 HTTP Version Not Supported</h1></center>" CRLF
+"<span class=\"big-text\">505</span>" CRLF
+"<span class=\"small-text\">HTTP Version Not Supported</span>" CRLF
 ;
 
-
 static char ngx_http_error_507_page[] =
 "<html>" CRLF
-"<head><title>507 Insufficient Storage</title></head>" CRLF
+"<head>" CRLF
+"<title>507</title>" CRLF
+"<style>" CRLF
+"html { color-scheme: light dark; }" CRLF
+"body { width: 35em; margin: 0 auto; font-family: Tahoma, Verdana, Arial, sans-serif; text-align: center; display: flex; flex-direction: column; justify-content: center; height: 100vh; }" CRLF
+"span { font-family: sans-serif; white-space: normal; max-width: 90%; display: block; margin: 0 auto; }" CRLF
+"span.big-text { font-size: 100px; font-weight: bold; margin-bottom: 20px; }" CRLF
+"span.small-text { font-size: 40px; font-weight: 100; }" CRLF
+"@media (max-width: 768px) { span.big-text { font-size: 70px; } span.small-text { font-size: 30px; } }" CRLF
+"@media (max-width: 480px) { span.big-text { font-size: 50px; } span.small-text { font-size: 20px; } }" CRLF
+"</style>" CRLF
+"</head>" CRLF
 "<body>" CRLF
-"<center><h1>507 Insufficient Storage</h1></center>" CRLF
+"<span class=\"big-text\">507</span>" CRLF
+"<span class=\"small-text\">Insufficient Storage</span>" CRLF
 ;
 
-
 static ngx_str_t ngx_http_error_pages[] = {
 
     ngx_null_string,                     /* 201, 204 */
@@ -680,18 +1035,10 @@
     ngx_uint_t    msie_padding;
     ngx_chain_t   out[3];
 
-    if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-        len = sizeof(ngx_http_error_full_tail) - 1;
-        tail = ngx_http_error_full_tail;
-
-    } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-        len = sizeof(ngx_http_error_build_tail) - 1;
-        tail = ngx_http_error_build_tail;
 
-    } else {
-        len = sizeof(ngx_http_error_tail) - 1;
-        tail = ngx_http_error_tail;
-    }
+    len = sizeof(ngx_http_error_tail) - 1;
+    tail = ngx_http_error_tail;
+
 
     msie_padding = 0;
 
diff --color -ruN nginx-1.26.2/src/http/v2/ngx_http_v2_filter_module.c nginx-1.26_changes/src/http/v2/ngx_http_v2_filter_module.c
--- nginx-1.26.2/src/http/v2/ngx_http_v2_filter_module.c	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/src/http/v2/ngx_http_v2_filter_module.c	2024-09-18 06:44:11.456169312 +0200
@@ -219,15 +219,9 @@
 
     if (r->headers_out.server == NULL) {
 
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            len += 1 + nginx_ver_len;
 
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            len += 1 + nginx_ver_build_len;
-
-        } else {
             len += 1 + sizeof(nginx);
-        }
+        
     }
 
     if (r->headers_out.date == NULL) {
@@ -423,45 +417,16 @@
 
     if (r->headers_out.server == NULL) {
 
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            ngx_log_debug1(NGX_LOG_DEBUG_HTTP, fc->log, 0,
-                           "http2 output header: \"server: %s\"",
-                           NGINX_VER);
-
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            ngx_log_debug1(NGX_LOG_DEBUG_HTTP, fc->log, 0,
-                           "http2 output header: \"server: %s\"",
-                           NGINX_VER_BUILD);
 
-        } else {
-            ngx_log_debug0(NGX_LOG_DEBUG_HTTP, fc->log, 0,
-                           "http2 output header: \"server: nginx\"");
-        }
+        ngx_log_debug0(NGX_LOG_DEBUG_HTTP, fc->log, 0,
+                           "http2 output header: \"server: hidden\"");
+        
 
         *pos++ = ngx_http_v2_inc_indexed(NGX_HTTP_V2_SERVER_INDEX);
 
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            if (nginx_ver[0] == '\0') {
-                p = ngx_http_v2_write_value(nginx_ver, (u_char *) NGINX_VER,
-                                            sizeof(NGINX_VER) - 1, tmp);
-                nginx_ver_len = p - nginx_ver;
-            }
 
-            pos = ngx_cpymem(pos, nginx_ver, nginx_ver_len);
-
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            if (nginx_ver_build[0] == '\0') {
-                p = ngx_http_v2_write_value(nginx_ver_build,
-                                            (u_char *) NGINX_VER_BUILD,
-                                            sizeof(NGINX_VER_BUILD) - 1, tmp);
-                nginx_ver_build_len = p - nginx_ver_build;
-            }
-
-            pos = ngx_cpymem(pos, nginx_ver_build, nginx_ver_build_len);
-
-        } else {
-            pos = ngx_cpymem(pos, nginx, sizeof(nginx));
-        }
+        pos = ngx_cpymem(pos, nginx, sizeof(nginx));
+        
     }
 
     if (r->headers_out.date == NULL) {
diff --color -ruN nginx-1.26.2/src/http/v3/ngx_http_v3_filter_module.c nginx-1.26_changes/src/http/v3/ngx_http_v3_filter_module.c
--- nginx-1.26.2/src/http/v3/ngx_http_v3_filter_module.c	2024-08-12 16:28:31.000000000 +0200
+++ nginx-1.26_changes/src/http/v3/ngx_http_v3_filter_module.c	2024-09-18 06:36:07.981760531 +0200
@@ -159,14 +159,7 @@
     clcf = ngx_http_get_module_loc_conf(r, ngx_http_core_module);
 
     if (r->headers_out.server == NULL) {
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            n = sizeof(NGINX_VER) - 1;
-
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            n = sizeof(NGINX_VER_BUILD) - 1;
-
-        } else {
-            n = sizeof("nginx") - 1;
+            n = sizeof("") - 1;
         }
 
         len += ngx_http_v3_encode_field_lri(NULL, 0,
@@ -340,17 +333,8 @@
     }
 
     if (r->headers_out.server == NULL) {
-        if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
-            p = (u_char *) NGINX_VER;
-            n = sizeof(NGINX_VER) - 1;
-
-        } else if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_BUILD) {
-            p = (u_char *) NGINX_VER_BUILD;
-            n = sizeof(NGINX_VER_BUILD) - 1;
-
-        } else {
-            p = (u_char *) "nginx";
-            n = sizeof("nginx") - 1;
+            p = (u_char *) "";
+            n = sizeof("") - 1;
         }
 
         ngx_log_debug2(NGX_LOG_DEBUG_HTTP, c->log, 0,
